setup.py

```diff
@@ -4,7 +4,7 @@
 
 setup(
     name='discussion-edx-platform-extensions',
-    version='1.1.9',
+    version='1.1.10',
```

Comment:
We already released 1.2.0, so the next one should be 1.2.1.

Reply:
Ah, right. My bad.